pkg/steps/template.go

```diff
@@ -46,6 +55,16 @@ func (s *templateExecutionStep) Inputs(ctx context.Context, dry bool) (api.Input
 	return nil, nil
 }
 
+func getOrgRepoFromVars(jobspec downwardapi.JobSpec) string {
```

Comment:
You have the job spec in the template step already, just use the values from that?

Reply:
I am using the jobspec from the template step already, [here](https://github.com/openshift/ci-tools/pull/77/files#diff-37ed7e4b3e5f7bb5d23118bff1f75bd9R106)